lib/internal/streams/readable.js

```diff
@@ -1239,7 +1239,7 @@ ObjectDefineProperties(Readable.prototype, {
     }
   },
 
-  readableErrored: {
+  errored: {
```

Comment:
Do the `readableErrored` and `writableErrored` need to go through a deprecation cycle first? Those should likely be runtime deprecated aliases, yes?

Reply:
Not really. They haven’t been in any release.